### PR TITLE
HDDS-4586. Mismatch in prefix of DN and SCM http.auth.type config key

### PR DESCRIPTION
**Pull request by flirmnave**
## What changes were proposed in this pull request?

Modify the document to match the corresponding config keys.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4586

## How was this patch tested?

Local hugo

### CODE REVIEW

File: hadoop-hdds/docs/content/security/SecuringOzoneHTTP.md [hunk 1]
```diff
@@ -128,17 +128,17 @@ change ozone.recon.http.auth.simple.anonymous_allowed to true.
 ### Enable SIMPLE authentication for SCM HTTP
 Property| Value
 -----------------------------------|-----------------------------------------
-ozone.scm.http.auth.type | simple
-ozone.scm.http.auth.simple.anonymous_allowed | false
+hdds.scm.http.auth.type | simple
+hdds.scm.http.auth.simple.anonymous_allowed | false
 
 If you don't want to specify the user.name in the query string parameter, 
 change ozone.scm.http.auth.simple.anonymous_allowed to true.
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/security/SecuringOzoneHTTP.md:135]:
```suggestion
change hdds.scm.http.auth.simple.anonymous_allowed to true.
```

File: hadoop-hdds/docs/content/security/SecuringOzoneHTTP.md [hunk 2]
```diff
@@ -128,17 +128,17 @@ change ozone.recon.http.auth.simple.anonymous_allowed to true.
...
 ### Enable SIMPLE authentication for DATANODE HTTP
 Property| Value
 -----------------------------------|-----------------------------------------
-ozone.datanode.http.auth.type | simple
-ozone.datanode.http.auth.simple.anonymous_allowed | false
+hdds.datanode.http.auth.type | simple
+hdds.datanode.http.auth.simple.anonymous_allowed | false
 
 If you don't want to specify the user.name in the query string parameter, 
 change ozone.datanode.http.auth.simple.anonymous_allowed to true.
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/security/SecuringOzoneHTTP.md:144]:
```suggestion
change hdds.datanode.http.auth.simple.anonymous_allowed to true.
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @flirmnave for taking care of this.  There are other mentions in the text.

**Comment by flirmnave** [comment]
Thanks @adoroszlai for the reminder.

**Review by xiaoyuyao — APPROVED** [review]
LGTM, +1.

**Comment by flirmnave** [comment]
Thanks @adoroszlai @xiaoyuyao for review and comment.

**Pull request merged**